js/apps/system/_admin/aardvark/APP/frontend/js/arango/arango.js

```diff
@@ -1066,9 +1066,10 @@
       var dlType;
       if (type === 'csv') {
         dlType = 'text/csv; charset=utf-8';
-      }
-      if (type === 'json') {
+      } else if (type === 'json') {
         dlType = 'application/json; charset=utf-8';
+      } else if (type === 'text') {
+        dlType = 'text/html; charset=utf8';
```

Comment:
If this is for our metrics out, I think the correct Content-Type would be `text/plain`, not `text/html`.

Reply:
Done, ty.